ledger/ledger-on-memory/BUILD.bazel

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) 2019 The DAML Authors. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+load(
+    "//bazel_tools:scala.bzl",
+    "da_scala_binary",
+    "da_scala_library",
+    "da_scala_test",
+)
+load("//ledger/ledger-api-test-tool:conformance.bzl", "conformance_test")
+
+compile_deps = [
```

Comment:
It is recommended to explicitly enumerate dependencies in the respective build targets to allow easier maintenance using `buildozer`.

Reply:
lol I love that name :tractor:

Reply:
Done as part of merging `master` in.